straxen/contexts.py

```diff
@@ -46,7 +46,7 @@
     # Clustering/classification parameters
     s1_max_rise_time=100,
     s2_xy_correction_map=("CMT_model", ('s2_xy_map', "ONLINE"), True),
-    elife_file=("elife_model", "ONLINE",True),
+    elife_conf=("elife", "ONLINE", True),
```

Comment:
It is tricky, since there is not a model in CMT but actual measurement values, so I will leave this discussion for when we revise all strax.Options() to make sure everything is consistent, then we can look at the name conventions of CMT.

Reply:
The Online version might be, but version1, version2 etc won't be.

Reply:
we don't know, they might come from Kr83 measurement